docs/topics/auth/default.txt

```diff
@@ -354,23 +354,34 @@ If you have an authenticated user you want to attach to the current session
                 # Return an 'invalid login' error message.
                 ...
 
-.. admonition:: Calling ``authenticate()`` first
+    .. versionchanged:: 1.10
```

Comment:
This annotation should appear below the description of the new behavior. I think I'd retain some of the old text and say something like "In older versions, if you manually logged a user, you must...."